luatest/server.lua

```diff
@@ -163,8 +163,8 @@ function Server:initialize()
         local max_unix_socket_path = {linux = 107, other = 103}
         local system = os.execute('[ $(uname) = Linux ]') == 0 and 'linux' or 'other'
         if parsed_net_box_uri.unix:len() > max_unix_socket_path[system] then
-            error(('Net box URI must be <= max Unix domain socket path length (%d chars)')
-                :format(max_unix_socket_path[system]))
+            error(('Unix domain socket path length must be not greater than ' ..
+                   '%d chars'):format(max_unix_socket_path[system]))
```

Comment:
While we're here, I would add the the problematic path into the message to make it easier to find a place in the code that triggers the problem.

Reply:
What is the reason for changing the error message?

Reply:
Old message was about whole URI, but only Unix socket path have the limitation.

Reply:
Net box URI may contain creds, got it, thanks.

Reply:
Fixed.